scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -926,6 +926,23 @@ public void removeJob(final List<JobId> jobIds, final long removedTime, final bo
         });
     }
 
+    public List<JobId> getJobsByFinishedTime(long olderThan) {
+        return executeReadOnlyTransaction(session -> {
+            logger.info("Identifying jobs by finished time");
+
+            Query query;
+            query = session.getNamedQuery("loadJobsWithFinishedTime")
+                           .setParameter("maxFinishedTime", olderThan)
+                           .setReadOnly(true);
```

Comment:
Why not simply:
`Query query = session.getNamedQuery("loadJobsWithFinishedTime")
                           .setParameter("maxFinishedTime", olderThan)
                           .setReadOnly(true);`

Reply:
right